libp2p/Host.h

```diff
@@ -318,6 +318,8 @@ class Host: public Worker
 	/// Peers we try to connect regardless of p2p network.
 	std::set<NodeID> m_requiredPeers;
 	Mutex x_requiredPeers;
+	/// returns true if a member of m_requiredPeers
+	bool isRequiredPeer(NodeID const&); // cannot be const because of Mutex
```

Comment:
the usual thing to do in this case is to make mutex `mutable` and leave the method `const`

Reply:
Done.